addOns/automation/src/main/java/org/zaproxy/addon/automation/AbstractAutomationTest.java

```diff
@@ -93,6 +95,10 @@ public void reset() {
         this.passed = null;
     }
 
+    public LinkedHashMap<?, ?> getTestData() {
```

Comment:
Sorry, noy sure I understand?

Reply:
I think i am overthinking, but my point was that the caller to this method doesn't needs to know the concrete implementation. It can be written over the interface.